09. cli-and-http-request/clihelper/cliStart.go

```diff
@@ -0,0 +1,18 @@
+package clihelper
+
+type CmdRunner struct {
```

Comment:
Let's move all the structs to structs and this one to structs\StdInParamters.go @Nafrose